client/packages/config/src/config.ts

```diff
@@ -0,0 +1,21 @@
+import { Capacitor } from '@capacitor/core';
+
+// For production we get URL relative to web app
+const { port, hostname, protocol } = window.location;
+
+const isProductionBuild = process.env['NODE_ENV'] === 'production';
+
+const devServerURL = 'https://demo-open.msupply.org:8000'; // Demo - site URL
+// const devServerURL = 'http://localhost:8000'; // - default URL for the backend graphql server
+
+// For mobile always use https://localhost:8000'
```

Comment:
slightly troubled by the hard coding of port numbers too.. another config setting?

Reply:
I've added a comment b429a2444eb1663b93fa9f9c790e78cb652fe705, it's already hard coded in:

https://github.com/openmsupply/open-msupply/blob/66d9dce56f813cccea80cbf20ecd93c812652b9a/client/packages/android/app/src/main/java/org/openmsupply/client/MainActivity.java#L19